src/magql/resolver_factory.py

```diff
@@ -151,7 +151,24 @@ def resolve(self, parent, info, *args, **kwargs):
         return value
 
 
-class CamelResolver(Resolver):
+class ResultResolver:
+    """
+    Result Resolver retrieves the result key off of the payload object
+    :param parent: GraphQL Payload object
+    :param info: gql info dictionary
+    :return: The results field on the payload object
+    """
+
+    def __call__(self, parent, info, *args, **kwargs):
+        value = (
```

Comment:
If we didn't make the change elsewhere to `return {"result": resolved_value}`, couldn't this just be `return parent`? That way we don't require people to know they have to make a dict with a result key in resolvers.